tests/fixed_buf.rs

```diff
@@ -90,6 +90,47 @@ fn unregister_invalidates_checked_out_buffers() {
     });
 }
 
+#[test]
```

Comment:
This is very cool. Would you mind adding a few one line comments saying what the asserts are proving? I know what they should be proving, since it is called slicing, but a few comments can go a long way in getting the light bulb to go off for someone. Again. Slicing fixed buffers is way cool. I'm glad the uring driver folks thought of it and I'm glad this is going to support it.